donation-tracker/src/components/layout.tsx

```diff
@@ -159,7 +176,7 @@ const LayoutModule = (props: any) => {
                 open={Boolean(anchorElShare)}
                 onClose={handleCloseShareMenu}
               >
-                {shareLinks.map((share, index) => (
+                {shareLinks.map((share: any, index: number) => (
```

Comment:
I know we could improve the type here, but I went with any for now - to have no longer red squiggly lines in VS Code ;)

Reply:
that's strange, for me it works with showing the right types on mouse over in VS code. Maybe restart TS language server.